scrapers_next/ca/committees.py

```diff
@@ -287,43 +145,43 @@ def process_item(self, item):
             com = ScrapeCommittee(
                 name=comm_name, classification="committee", chamber="upper"
             )
-        com.add_source(self.source.url)
-        com.add_source(comm_url)
-        com.add_link(comm_url, note="homepage")
-        return ChooseType(com, source=URL(comm_url))
+
+        com.add_source(self.source.url, note="Committee List Page")
+        com.add_source(comm_url, note="Committee Detail Page")
+
+        return CommDetails(com, source=URL(comm_url))
 
 
 class AssemblyCommitteeList(HtmlListPage):
     source = URL("https://www.assembly.ca.gov/committees")
-
-    selector = CSS("div .block.block-views ul li", min_items=90)
+    selector = XPath(".//div[@class='views-field views-field-title']")
 
     def process_item(self, item):
         comm_name = CSS("a").match_one(item).text_content()
         comm_url = CSS("a").match_one(item).get("href")
 
-        # "https://jtlegbudget.legislature.ca.gov/sublegislativeanalyst" has no members
-        if comm_url == "https://jtlegbudget.legislature.ca.gov/sublegislativeanalyst":
-            self.skip()
+        # Joint Committees are being skipped to avoid duplicates
+        #  because they are grabbed during SenateCommitteeList()
+        if "joint" in comm_name.lower():
+            self.skip("Joint committees retrieved by SenateCommitteeList() object")
 
-        # Joint Committees are being skipped to avoid duplicates (they were already grabbed during SenateCommitteeList())
-        if comm_name.startswith("Joint Committee") or comm_name.startswith(
-            "Joint Legislative"
-        ):
-            self.skip()
-        elif comm_name.startswith("Subcommittee"):
-            parent_comm = item.getparent().getparent().getchildren()[0].text_content()
-            com = ScrapeCommittee(
-                name=comm_name,
-                classification="subcommittee",
-                chamber="lower",
-                parent=parent_comm,
+        if comm_name.startswith("Subcommittee"):
```

Comment:
Some Assembly Subcommittees are getting scraped with the Parent committee of `Subcommittee No. 1 on Health and Human Services`

Reply:
I have a feeling this was because of the selector change I made in my prior commit. However[ I just fixed the parent selection](https://github.com/openstates/openstates-scrapers/pull/4397/files#diff-d2282de25dd52de0c420d1d9e3ba2378a39f614b5b6ef2c0590624dbba51f483R170-R176) so we should be good now.